src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskCompleteByIdControllerTest.java

```diff
@@ -147,137 +115,6 @@ public void should_return_a_403_if_task_was_not_previously_assigned() {
 
     }
 
-    @Test
-    public void should_return_a_204_when_completing_a_task_by_id_with_restricted_role_assignment() {
-
-        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json", "processApplication");
-        taskId = taskVariables.getTaskId();
-        initiateTask(taskVariables, Jurisdiction.WA);
-
-        common.setupRestrictedRoleAssignmentForWA(taskVariables.getCaseId(), caseworkerCredentials.getHeaders());
-        given.iClaimATaskWithIdAndAuthorization(
-            taskId,
-            caseworkerCredentials.getHeaders(),
-            HttpStatus.NO_CONTENT
-        );
-        Response result = restApiActions.post(
-            ENDPOINT_BEING_TESTED,
-            taskId,
-            caseworkerCredentials.getHeaders()
-        );
-
-        result.then().assertThat()
-            .statusCode(HttpStatus.NO_CONTENT.value());
-
-        assertions.taskVariableWasUpdated(taskVariables.getProcessInstanceId(), "taskState", "completed");
-        assertions.taskStateWasUpdatedInDatabase(taskId, "completed", caseworkerCredentials.getHeaders());
-
-        common.cleanUpTask(taskId);
-
-    }
-
-    @Test
-    public void should_return_a_401_when_the_user_did_not_have_any_roles() {
-
-        TestVariables taskVariables = common.setupWATaskAndRetrieveIds("requests/ccd/wa_case_data.json", "processApplication");
-        taskId = taskVariables.getTaskId();
-        initiateTask(taskVariables, Jurisdiction.WA);
-
-        Response result = restApiActions.post(
-            ENDPOINT_BEING_TESTED,
-            taskId,
-            caseworkerCredentials.getHeaders()
-        );
-
-        result.then().assertThat()
-            .statusCode(HttpStatus.UNAUTHORIZED.value())
-            .contentType(APPLICATION_JSON_VALUE)
-            .body("timestamp", lessThanOrEqualTo(ZonedDateTime.now().plusSeconds(60)
-                .format(DateTimeFormatter.ofPattern(DATE_TIME_FORMAT))))
-            .body("error", equalTo(HttpStatus.UNAUTHORIZED.getReasonPhrase()))
-            .body("status", equalTo(HttpStatus.UNAUTHORIZED.value()))
-            .body("message", equalTo("User did not have sufficient permissions to perform this action"));
-
-        common.cleanUpTask(taskId);
-
-    }
-
-    @Test
-    public void should_return_a_204_and_retrieve_a_task_by_id_jurisdiction_location_and_region_match() {
```

Comment:
should we keep this? Can see IT.

Reply:
There are many IT that tests role assignment verification for location and region match. For this spesific tests, there are few tests in CftQueryServiceCompleteTaskTest